### PR TITLE
[INTERNAL, FEATURE] Clean up of folder and add openMS support

### PR DESCRIPTION
**Pull request by smehringer**
This PR removes the unnecessary old scripts and logfiles and introduces parsing of OpenMS logs.

@jpfeuffer Please add an ssh call for the openMS logs in Jenkins. I can then update the Jenkins script to also create the openMS report.

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Yes I added a command. The logs go to log_files/openms.
By the way, is the process_seqan_logfiles specific to seqan? Can we rename it to process_logfiles and use it twice. Maybe with an option to specify an output file?

**Comment by smehringer** [comment]
@jpfeuffer looks like it works now. The html page just has seqan statistics as a title...

**Comment by jpfeuffer** [comment]
@smehringer Yep looks good. There is another occurrence of SeqAn for that OpenMS specific column "who". Should probably be changed, too.

**Comment by jpfeuffer** [comment]
Ah also the plot titles sometimes.

File: scripts/create_report.R [hunk 1]
```diff
@@ -71,6 +71,11 @@ if(!require(RColorBrewer)) {
 	library(RColorBrewer, lib.loc=local.lib)
 }
 
+if(!require(yaml)) {
+        install.packages("yaml", repos = c('http://rforge.net', 'http://cran.rstudio.org'), type = 'source', lib=local.lib)
+        library(RColorBrewer, lib.loc=local.lib)
```
Comment (cbielow) [inline scripts/create_report.R]: should say `library(yaml, lib.loc=local.lib)` ?

Comment (smehringer) [reply scripts/create_report.R]: good catch thanks.

**Comment by jpfeuffer** [comment]
I think you don't need the script.basename for the render location because you added getWd in the beginning already.

**Comment by smehringer** [comment]
@jpfeuffer thanks for fixing the last things! :+1:

**Pull request merged**